build/changelog-generator/index.js

```diff
@@ -5,6 +5,7 @@
  *  Based on: https://github.com/GoogleChrome/lighthouse/blob/master/build/changelog-generator/
  */
 'use strict';
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
Can be removed

Reply:
Linter threw here so I don't think so

Reply:
Ok it might be an issue of the linter (we could upgrade later). This is not a ts file after all